cli/src/nonce.rs

```diff
@@ -95,16 +95,16 @@ impl NonceSubCommands for App<'_, '_> {
                 .arg(
                     Arg::with_name("nonce_account_keypair")
                         .index(1)
-                        .value_name("NONCE_ACCOUNT")
+                        .value_name("KEYPAIR")
```

Comment:
This should be PUBKEY

Reply:
Even though the option name ends with `_keypair`?

Reply:
I think that variable name is likely a relic from before nonce_authority was implemented, and the account needed to be the signer. Anyway, this: 

> `--nonce-account-keypair` should be `--nonce-account-pubkey`

If you want to make that change, great! Otherwise I'll work that in to my fix.

Reply:
Fixed in fa66062, thanks